main.go

```diff
@@ -133,26 +43,49 @@ func main() {
 				log.Fatalf("couldn't close file %v, %v\n", *destFile, err)
 			}
 		}()
-
 		dst = dstFile
 	}
 
-	w := bufio.NewWriterSize(dst, *writeBufferSize)
+	data, err := mm.Map(src, mm.RDONLY, 0)
+	if err != nil {
+		log.Fatalf("couldn't mmap file: %v\n", err)
+	}
+	if len(data) <= 0 {
+		return
+	}
+
+	var str []string
 
+	// identify lines in data
+	p := 0
+	for i := 0; i < len(data); i++ {
+		if data[i] == '\n' {
+			str = append(str, sixb.BtS(data[p:i+1]))
+			p = i + 1
+		}
+	}
+	if p < len(data) {
+		str = append(str, sixb.BtS(data[p:])) // last line
+	}
+
+	// in-place concurrent sorting, no extra memory, 4 goroutines
+	sorty.Mxg = 4
```

Comment:
should this be `	sorty.Mxg = uint32(runtime.NumCPU())`?

Reply:
if you want to use all cores, yes.